src/workerd/api/streams/queue.c++

```diff
@@ -54,10 +54,6 @@ ValueQueue::QueueEntry ValueQueue::QueueEntry::clone(jsg::Lock& js) {
   return QueueEntry { .entry = entry->clone(js) };
 }
 
-void ValueQueue::QueueEntry::visitForGc(jsg::GcVisitor& visitor) {
```

Comment:
It is my ignorance, but without a commit message, I'm not sure what's being solved here.

Could you add a brief description to the commit message that explains why the `visitForGc` isn't necessary?